jobs/types/audit_errors.py

```diff
@@ -180,6 +180,15 @@ def __init__(self, model):
             feconf.PERIOD_TO_MARK_MODELS_AS_DELETED.days)
 
 
+class ModelIncorrectKeyError(BaseAuditError):
+    """Error class for incorrect key in PendingDeletionRequestModel."""
+
+    def __init__(self, model, incorrect_keys):
+        super(ModelIncorrectKeyError, self).__init__(model)
+        self.message = 'Entity id %s: contains keys %s that are not allowed' % (
+            model.id, incorrect_keys)
```

Comment:
`'Entity id %s:` part is not required here. As you can see the ID is already thrown in the error. Ex: `ModelIncorrectKeyError in PendingDeletionRequestModel(id='test')`

Reply:
thanks done!